app/scripts/templates/settings/clients.mustache

```diff
@@ -18,7 +18,7 @@
         {{#clients}}
           <li class="client {{type}} client-{{clientType}} {{#isCurrentDevice}}client-current{{/isCurrentDevice}}"
             data-name="{{name}}" id="{{id}}">
-            <div class="client-name" title="{{name}}">{{name}}</div>
+            <div class="client-name" title="{{name}} - {{scope}}">{{name}}</div>
```

Comment:
@divyabiyani one other to handle here is: Devices won't have scope so then the title for the Device will be: `title="vlad's device - ".

We do not want the empty `-`, so we need a way to handle that.